### PR TITLE
Revamp user tutorial

### PR DESCRIPTION
**Pull request by CBroz1**

### CODE REVIEW

File: mkdocs.yaml [hunk 1]
```diff
@@ -89,6 +90,12 @@ markdown_extensions:
         - name: mermaid
           class: mermaid
           format: !!python/name:pymdownx.superfences.fence_code_format
+  - pymdownx.tabbed:
+      alternate_style: true
+  - pymdownx.highlight:
+      linenums: true
+  - pymdownx.inlinehilite
+  - pymdownx.snippets
```
Comment (guzman-raphael) [inline mkdocs.yaml:98]: @CBroz1 Could you elaborate on what these mkdocs features do for those not familiar? Perhaps we can add it to the PR description or as inline comments.

**Review by guzman-raphael — CHANGES_REQUESTED** [review]

File: mkdocs.yaml (hunk 1 above)
Comment (CBroz1) [reply mkdocs.yaml]: Certainly - Just added inline comments, and commented out items I not actually used in this PR

**Review by dimitri-yatsenko — DISMISSED** [review]

**Review by kabilar — CHANGES_REQUESTED** [review]
Please pull from upstream and resolve conflicts.

**Comment by CBroz1** [comment]
Thanks for reviewing, @kabilar . I have fetched from upstream and resolved conflicts

**Review by kabilar — APPROVED** [review]
Thanks @CBroz1!  Great work.

As we are flushing out the `Tutorials` page for each Element, let's make sure that it integrates well with this User Guide.

**Comment by kabilar** [comment]
@guzman-raphael Can you please approve and then I will merge?  Currently blocked by branch protection rules.  Thank you.

**Review by guzman-raphael — APPROVED** [review]

**Pull request merged**